scripts/repo_health.sh

```diff
@@ -6,6 +6,7 @@ hab pkg install -b core/go core/git core/ruby core/jq-static core/shellcheck cor
 
 echo "Checking Go Dependencies"
 go mod verify
+go mod tidy -v
```

Comment:
I think the problem with this is that it'll download all the dependencies and thus open us up to more network flakiness